crates/bitwarden-crypto/src/enc_string/asymmetric.rs

```diff
@@ -11,11 +11,16 @@ use crate::{
     rsa::encrypt_rsa2048_oaep_sha1,
     AsymmetricCryptoKey, AsymmetricEncryptable, KeyDecryptable,
 };
-
 // This module is a workaround to avoid deprecated warnings that come from the ZeroizeOnDrop
 // macro expansion
 #[allow(deprecated)]
 mod internal {
+    #[cfg(feature = "wasm")]
+    #[wasm_bindgen::prelude::wasm_bindgen(typescript_custom_section)]
+    const TS_CUSTOM_TYPES: &'static str = r#"
+    export type AsymmetricEncString = string;
+    "#;
```

Comment:
We have a `uniffi_support.rs` file, would it make sense to add something similar for these custom logics? Or should we encourage lifting in the uniffi logic to where it's needed.

Reply:
I added `custom_types.rs` which is inspired by that uniffi file, but I think adding the types where they are defined is a preferred approach. It's just that we sometimes use external type and so I needed both approaches